robottelo/host_helpers/cli_factory.py

```diff
@@ -717,13 +724,15 @@ def _setup_org_for_a_rh_repo(self, options=None):
             env_id = self.make_lifecycle_environment({'organization-id': org_id})['id']
         else:
             env_id = options['lifecycle-environment-id']
-        # If manifest does not exist, clone and upload it
-        if len(self._satellite.cli.Subscription.exists({'organization-id': org_id})) == 0:
-            with clone() as manifest:
-                try:
-                    self._satellite.upload_manifest(org_id, manifest.content)
-                except CLIReturnCodeError as err:
-                    raise CLIFactoryError(f'Failed to upload manifest\n{err.msg}')
+        # Clone manifest and upload it
```

Comment:
It looks to me like this change will cause tests that pass `force_use_cdn=True` to the `setup_org_for_a_rh_repo` method to always use cloned manifests. Is that intended?

Reply:
great catch! I'm not sure how that conditional was deleted, but i don't see a reason why it should have been. put back in place in the latest change.